### PR TITLE
[18Uruguay] Do not ship more than ship capacity for rptla

### LINKED ISSUE #11140: 18Uruguay - RTPLA liquidation shipping phase does not follow the printed rules
https://18xx.games/game/172678?action=711

When the RPTLA does its final shipping phase as part of being liquidated, it is ignoring the shipping capacity of the ships it owns.  As I understand it, a 4-ship can carry 2 goods, and a 5-ship can carry 3 goods.  But here, the RPTLA was allowed to ship 8 goods (instead of 5) in its final shipping phase.  The rules provide as follows:

10.2.1 RPTLA - Final Export and Payout
Move all cubes in Ports to the Frigorificos Anglo-box.
Then export cubes from the Frigorificos Anglo-box
to the European Market equalling the Total Shipping Capacity and calculate the Export Revenue, 50
per cube exported. 

In the final export phase, the RPTLA still is limited to the total shipping capacity.  This bug has been encountered in all of the 18Uruguay games I have played that have reached liquidation phase.

### PR DESCRIPTION
**Pull request by patrikolesen**
Fixes #11140

## Before clicking "Create"

- [X] Branch is derived from the latest `master`
- [ ] Add the `pins` or `archive_alpha_games` label if this change will break existing games
- [X] Code passes linter with `docker compose exec rack rubocop -a`
- [X] Tests pass cleanly with `docker compose exec rack rake`

### CODE REVIEW

File: lib/engine/game/g_18_uruguay/game.rb [hunk 1]
```diff
@@ -535,6 +545,10 @@ def operating_order
 
           super.reject { |c| c == @rptla }.append(@rptla)
         end
+
+        def ship_capacity(train)
+          SHIP_CAPACITY[train.name.partition('+')[0]]
```
Comment (ollybh) [inline lib/engine/game/g_18_uruguay/game.rb:550]: What's the call to partition doing here? Isn't `train.name` going to be something like `Ship 3` with no plus sign in it?

File: lib/engine/game/g_18_uruguay/step/route_rptla.rb [hunk 2]
```diff
@@ -55,13 +55,9 @@ def choices
             choices
           end
 
-          def ship_capacity(train)
-            SHIP_CAPACITY[train.name.partition('+')[0]]
```
Comment (ollybh) [inline lib/engine/game/g_18_uruguay/step/route_rptla.rb:59]: As you're removing this method from this class, the `SHIP_CAPACITY` constant at the start of this file is no longer needed.

**Review by ollybh — CHANGES_REQUESTED** [review]

File: lib/engine/game/g_18_uruguay/game.rb (hunk 1 above)
Comment (patrikolesen) [reply lib/engine/game/g_18_uruguay/game.rb]: You are correct this was legacy code, removed

Also added nil check in ship_capacity

File: lib/engine/game/g_18_uruguay/step/route_rptla.rb (hunk 2 above)
Comment (patrikolesen) [reply lib/engine/game/g_18_uruguay/step/route_rptla.rb:59]: Removed

**Review by ollybh — APPROVED** [review]

**Pull request merged**